sentry-core/src/main/java/io/sentry/core/Session.java

```diff
@@ -96,12 +96,12 @@ public Session(
     this(
         State.Ok,
         DateUtils.getCurrentDateTime(),
-        null,
+        DateUtils.getCurrentDateTime(),
```

Comment:
so session started also has a timestamp otherwise the server would assume current time